src/signet.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2019-2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_SIGNET_H
+#define BITCOIN_SIGNET_H
+
+#if defined(HAVE_CONFIG_H)
+#include <config/bitcoin-config.h>
+#endif
+
+#include <consensus/params.h>
+
+#include <stdint.h>
+
+class CBlock;
+class CScript;
+class uint256;
+
+extern CScript g_signet_blockscript;
+
+constexpr uint8_t SIGNET_HEADER[4] = {0xec, 0xc7, 0xda, 0xa2};
```

Comment:
(I wonder sometimes why these constants are often not used... maybe to avoid including `script.h` or because everyone knows them by heart)
```diff
 #include <consensus/params.h>
+#include <script/script.h>
 
-constexpr uint8_t SIGNET_HEADER[4] = {0xec, 0xc7, 0xda, 0xa2};
+constexpr uint8_t SIGNET_HEADER[4] = {0xec, 0xc7, 0xda, OP_GREATERTHANOREQUAL};
```